### PR TITLE
1122 docker hub

### PR DESCRIPTION
**Pull request by sibsmc**
changes required for build to work in docker hub

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
       - POSTGRES_USER=mentoruser
 
     ports:
-      - '3000:3000'
+      - "3000:3000"
     volumes:
       - ./server/:/app
     tmpfs:
       - /app/tmp
   my-app:
```
Comment (diwanow) [inline docker-compose.yml:20]: Will you change my-app, please, to something better reflecting what it is, e.g. web, ui, frontend or something of sorts? It just annoys my inner nazi :D It is ok to have another pr for that, up to you.

**Review by diwanow — APPROVED** [review]
Great!

File: docker-compose.yml (hunk 1 above)
Comment (sibsmc) [reply docker-compose.yml:20]: Yeah I also think it should have a better name, I'll change the name of the react docker container to ui. 
Do you think we should change the folder structure?

Comment (diwanow) [reply docker-compose.yml]: Yes, then my mind will relax :) hopefully it won't be too much trouble

**Pull request merged**